sympy/physics/continuum_mechanics/tests/test_beam.py

```diff
@@ -470,16 +470,20 @@ def test_apply_support():
                 + 10*SingularityFunction(x, 4, 3)/3)/(E*I))
 
     b = Beam(30, E, I)
-    b.apply_support(10, "pin")
-    b.apply_support(30, "roller")
+    p0 = b.apply_support(10, "pin")
+    p1 = b.apply_support(30, "roller")
     b.apply_load(-8, 0, -1)
     b.apply_load(120, 30, -2)
-    R_10, R_30 = symbols('R_10, R_30')
-    b.solve_for_reaction_loads(R_10, R_30)
+    b.solve_for_reaction_loads(p0, p1)
     assert b.slope() == (-4*SingularityFunction(x, 0, 2) + 3*SingularityFunction(x, 10, 2)
             + 120*SingularityFunction(x, 30, 1) + SingularityFunction(x, 30, 2) + Rational(4000, 3))/(E*I)
     assert b.deflection() == (x*Rational(4000, 3) - 4*SingularityFunction(x, 0, 3)/3 + SingularityFunction(x, 10, 3)
             + 60*SingularityFunction(x, 30, 2) + SingularityFunction(x, 30, 3)/3 - 12000)/(E*I)
+    R_10 = Symbol('R_10')
+    R_30 = Symbol('R_30')
+    assert p0 == R_10
+    assert b.reaction_loads == {R_10: 6, R_30: 2}
+    assert b.reaction_loads[p0] == 6
```

Comment:
Can you also add a test for apply_fixed so that it tests returning two values also?